cpp/ql/src/semmle/code/cpp/rangeanalysis/RangeAnalysis.qll

```diff
@@ -161,7 +161,11 @@ private IRGuardCondition boundFlowCond(ValueNumber vn, NonPhiOperand bound, int
 {
   exists(int d |
     result.comparesLt(vn.getAUse(), bound, d, upper, testIsTrue) and
-    // strengthen from x < y to x <= y-1
+    /*
```

Comment:
A minor style comment: I'd go with `//`-style comments for in-predicate comments, and leave multiline `/* .. */` for top-level things.  (This is also what the autoformatter currently expects, so this would add a spurious newline when autoformatted)

Reply:
Agreed. In-predicate comments should be easy to ignore and should therefore not stand out visually.